src/presentation/components/TopBar/TopBar.module.scss

```diff
@@ -0,0 +1,29 @@
+@import '../../../app/styles/base/settings';
+
+.topBar {
+	margin: 0 0.8rem;
+
+	&__item {
+		cursor: pointer;
+		margin-top: 2rem !important;
+	}
+
+	&__button {
+		background-color: $mid-grey !important;
+		border-radius: 5rem !important;
+		color: white !important;
+		font-size: $font-size-small !important;
+		font-weight: $font-weight-light !important;
+		padding: 0.2 !important;
+		text-transform: none !important;
+		width: 7rem;
+
+		&:hover {
+			background-color: $light-grey !important;
```

Comment:
Are all of these `!important` needed here?

Reply:
This is a style for Material UI Buttons. In order to override the css of a material ui component the !important is needed when it comes to properties already set like the backgroundColor, fonts, etc.

Reply:
You don't need to use important to override material ui styles, reach out with me and I'll help you with that.